packages/tailwindcss/src/utilities.ts

```diff
@@ -515,10 +501,8 @@ export function createUtilities(theme: Theme) {
     ['left', 'left'],
   ] as const) {
     staticUtility(`${name}-auto`, [[property, 'auto']])
-    utilities.static(`${name}-full`, (candidate) => {
-      let value = candidate.negative ? '-100%' : '100%'
-      return [decl(property, value)]
-    })
+    utilities.static(`${name}-full`, () => [decl(property, '100%')])
+    utilities.static(`-${name}-full`, () => [decl(property, '-100%')])
```

Comment:
For consistency I think these should also use `staticUtility`:

```suggestion
    staticUtility(`${name}-full`, () => [property, '100%'])
    staticUtility(`-${name}-full`, () => [property, '-100%'])
```